packages/blade/src/utils/index.ts

```diff
@@ -15,3 +15,5 @@ export * from './makeSize';
 // https://github.com/razorpay/blade/pull/1193#issuecomment-1609122359
 export * from './toTitleCase';
 export * from './usePrevious';
+
+export { default as useTheme, ThemeContext } from '~components/BladeProvider/useTheme';
```

Comment:
don't think so we shall have interdependencies like this. we wanted to keep them separate hence separate packages. Its okay to have component specific hooks to be imported from components. also utils for meant for things which people can use generally but useTheme can't be used generally without blade's context